CMakeLists.txt

```diff
@@ -0,0 +1,180 @@
+SET(PACKAGE osm2pgsql)
+SET(PACKAGE_NAME osm2pgsql)
+
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.7)
+
+PROJECT(osm2pgsql)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+
+SET(DATA_DIR \".\")
+
+OPTION(BUILD_TESTS "Build with tests" OFF)
+
+if("${PROJECT_SOURCE_DIR}" STREQUAL "${PROJECT_BINARY_DIR}")
+  message(FATAL_ERROR "In-source builds are not allowed, please use a separate build directory like `mkdir biuld && cd build && cmake ..`")
+endif()
+
+execute_process (COMMAND git describe OUTPUT_VARIABLE PACKAGE_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
What's the git describe for? It should build with just the download, not a git checkout